frontend/public/package.json

```diff
@@ -74,8 +74,8 @@
     "postcss": "8.4.38",
     "postcss-loader": "6.2.1",
     "posthog-js": "^1.75.4",
-    "prettier-eslint": "9.0.2",
-    "prettier-eslint-cli": "5.0.1",
+    "prettier-eslint": "^16.3.0",
+    "prettier-eslint-cli": "^8.0.1",
```

Comment:
In order to get the `fmt` check passing, I had to update prettier-eslint (IMO we should use `prettier` itself and not prettier-eslint, but I've left that for another PR.)

The newer version of prettier-eslint uses prettier v3, which has some different formatting opinions.